### PR TITLE
Celery options

### PR DESCRIPTION
**Pull request by omerfeinberg**
This branch allows usage of options available for celery tasks: priority, serializer, compression, etc.
 
Great job with this package! it performs incredibly well

### CODE REVIEW

**Comment by brabiega** [comment]
Hello! Thanks a lot for this contribution. I'll try to take a look tomorrow or the day after. Looks quite good!

File: celery_dyrygent/workflows/workflow.py
Comment (brabiega) [inline celery_dyrygent/workflows/workflow.py]: Perhaps it would be better to use
```
def apply_async(self, options=None)
```
So in the future it would be possible to pass some additional arguments.
What do you think?

File: README.md
Comment (brabiega) [inline README.md]: I'm wondering if it would be better to specify options directly.
```
wf.apply_async(options=dict(priority=8))
```

**Review by brabiega — COMMENTED** [review]
Just a question about passing options to wf.apply_async. See comments

File: celery_dyrygent/workflows/workflow.py
Comment (omerfeinberg) [reply celery_dyrygent/workflows/workflow.py]: Thanks for the quick feedback man! 
So regarding both your comments, that's pretty much the only thing I wasn't sure about. I think we should support apply_async with the same signature as celery, which means using **kwargs. I don't think it should hurt us in the future while making changes, but I'm not 100% about that, so whatever you think :)

(Note that while initializing a Workflow object I do use options=dict(...), so your suggestion has the advantage of uniformity)

Comment (brabiega) [reply celery_dyrygent/workflows/workflow.py]: I know celery uses **kwargs, but I don't think that's a good design decision.
Let's go with options=dict() :)

Comment (omerfeinberg) [reply celery_dyrygent/workflows/workflow.py]: no problem, changed :)

Comment (omerfeinberg) [reply celery_dyrygent/workflows/workflow.py]: Hey man, waiting for your approval :)

Comment (brabiega) [reply celery_dyrygent/workflows/workflow.py]: Sure thing. 
I'd like to merge https://github.com/ovh/celery-dyrygent/pull/6 first.
And then release both changes under 0.8.0.
Besides I'm testing your changes right now :)

Edit:
changes required due to conflict are minimal

Comment (omerfeinberg) [reply celery_dyrygent/workflows/workflow.py]: Cool, they were indeed minimal.
I changed custom_payload to use the straight_serialzible default mechanism, I think it's neater.
No other changes

**Review by brabiega — APPROVED** [review]
Looks good. I'll just do some final tests tomorrow morning.

File: celery_dyrygent/workflows/workflow.py
Comment (brabiega) [reply celery_dyrygent/workflows/workflow.py]: Thx. The straight_serialzible default mechanism is pretty cool.

**Pull request merged**
